syft/messaging/promise.py

```diff
@@ -0,0 +1,155 @@
+"""This file contains the object we use to tell another worker that they will
```

Comment:
What is the exact motivation of putting this in messaging and not in promiseTensor?
Do we have reason to believe this will be reused by other types?

Reply:
I think we could indeed have promise for objects that are not tensors, maybe for instance strings like we plan to have with SyferText